tasks/src/unconditional-image-generation/data.ts

```diff
@@ -0,0 +1,55 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "NFT images dataset of 'cryptopunks' for unconditional image generation.",
+			id:          "huggingnft/cryptopunks",
+		},
+		{
+			description: "NFT images dataset of 'alpacadabraz' for unconditional image generation.",
+			id:          "huggingnft/alpacadabraz",
+		},
+	],
+	demo: {
+		inputs: [
+			{
+				label:   "Input",
+				content: "",
+				type:    "text",
+			},
+		],
+		outputs: [
+			{
+				filename: "unconditional-image-generation-output.jpeg",
+				type:     "img",
+			},
+		],
+	},
+	metrics: [
+		{
+			description: "The inception score (IS) evaluates the quality of image generative models by using the output of a pre-trained classifier (InceptionV3). It measures that the generated images are diverse (the model predictions are evenly distributed across all possible labels) and distinct (the model confidently predicts a single label for each image).",
+			id:          "Inception score (IS)",
+		},
+		{
+			description: "The Fréchet Inception Distance (FID) evaluates the quality of images created by a generative model by calculating the distance between feature vectors for real and generated images.",
+			id:          "Frećhet Inception Distance (FID)",
+		},
+	],
+	models: [
+		{
+			description: "LightWeight GAN model for unconditional generation.",
```

Comment:
I found these: https://huggingface.co/google/ddpm-celebahq-256 https://huggingface.co/google/ddpm-cifar10-32 especially models trained datasets like CIFAR are quite off the shelf and useful. What do you think? 🙂

Reply:
I agree :)! I am changing the models to these ones.